src/FormUser.js

```diff
@@ -0,0 +1,262 @@
+import * as React from 'react';
+
+class FormUser extends React.Component{
+    
+    constructor(){
+        super();
+        this.state = { users: [], nameText: '', lastNameText: '', emailText: '', phoneNumberText: '', countryText: '', addressText: '', shown: true};
+        this._handleChangeFirstName = this._handleChangeFirstName.bind(this);
+        this._handleChangeLastName = this._handleChangeLastName.bind(this);
+        this._handleChangeCountry = this._handleChangeCountry.bind(this);
+        this._handleChangeAddress = this._handleChangeAddress.bind(this);
+        this._handleChangeEmail = this._handleChangeEmail.bind(this);
+        this._handleChangePhoneNumber = this._handleChangePhoneNumber.bind(this);
+        this._handleClick = this._handleClick.bind(this);
+        this._handleClickDelete = this._handleClickDelete.bind(this);
+        this._handleClickEdit = this._handleClickEdit.bind(this);
+        this._handleClickSaveEdit = this._handleClickSaveEdit.bind(this);
+    }
+
+    /*toggle() {
+		this.setState({
+			shown: !this.state.shown
+		});
+	}*/
+
+    _handleChangeFirstName(event){
+        this.setState({
+            nameText: event.target.value
+        });
+    }
+
+    _handleChangeLastName(event){
+        this.setState({
+            lastNameText: event.target.value
+        });
+    }
+
+    _handleChangeCountry(event){
+        this.setState({
+            countryText: event.target.value
+        });
+    }
+
+    _handleChangeAddress(event){
+        this.setState({
+            addressText: event.target.value
+        });
+    }
+
+    _handleChangeEmail(event){
+        this.setState({
+            emailText: event.target.value
+        });
+    }
+
+    _handleChangePhoneNumber(event){
+        this.setState({
+            phoneNumberText: event.target.value
+        });
+    }
+
+    _handleClick(){
+        let newUser = {
+            id : Date.now(),
+            firstname : this.state.nameText,
+            lastname: this.state.lastNameText,
+            country: this.state.countryText,
+            address: this.state.addressText,
+            email: this.state.emailText,
+            phonenumber: this.state.phoneNumberText
```

Comment:
Rename: phonenumber property to 'phoneNumber' (camelCase)

Reply:
Done